WORKSPACE

```diff
@@ -4,9 +4,9 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
     name = "bazel_toolchains",
-    sha256 = "4fb3ceea08101ec41208e3df9e56ec72b69f3d11c56629d6477c0ff88d711cf7",
-    strip_prefix = "bazel-toolchains-3.6.0",
-    urls = ["https://github.com/bazelbuild/bazel-toolchains/releases/download/3.6.0/bazel-toolchains-3.6.0.tar.gz"],
+    sha256 = "1adf7a8e9901287c644dcf9ca08dd8d67a69df94bedbd57a841490a84dc1e9ed",
+    strip_prefix = "bazel-toolchains-5.0.0",
+    urls = ["https://github.com/bazelbuild/bazel-toolchains/archive/v5.0.0.tar.gz"],
```

Comment:
Replace with
```
    sha256 = "5b6eef36e1f627bf739d3212487c7c31e9f19ce68bcb4d672aa11be1bd6bd2ee",
    strip_prefix = "bazel-toolchains-5.1.0",
    urls = ["https://github.com/bazelbuild/bazel-toolchains/archive/v5.1.0.tar.gz"],
```

v5.1.0 contains some fixes for Windows

Reply:
done